charts/timed/values.yaml

```diff
@@ -5,22 +5,21 @@ backend:
   replicaCount: 1
   image:
     repository: adfinissygroup/timed-backend
-    tag: latest
+    tag: v0.12.1
     pullPolicy: IfNotPresent
   service:
     # Specifiy secret for authentication.
     # If not random token will be generated
     # secret:
     # Specify database url
     emailFrom: webmaster@chart-example.local
```

Comment:
We should move the Django settings somewhere else.

Reply:
which django settings?

Reply:
The `email*` settings aren't part of the service. They are Django settings. Maybe a top-level `timed` entry that contains all the Django shizzles makes sense.

Reply:
or move them to `.Values.backend.django` as django values only concern the backend?

Reply:
I moved them to `.Values.settings`